docs/contributing/GET_STARTED.md

```diff
@@ -24,8 +24,12 @@ melos bootstrap
 ## 3.　Read README.md
```

Comment:
![nits-badge](https://img.shields.io/badge/review-nits-orange.svg) 
This is beyond the scope of the pull request, but I would appreciate it if you could make the correction 🙏 
```suggestion
## 3. Read README.md
```